### ISSUE TITLE
How to use the filter export option ?

### ISSUE THREAD

**Issue by gsals** [dfahlander/Dexie.js#902]
Hello,

I have a really simple indexedDb table which contains objects of this type : _{name: "...", visible: true, data: ArrayBuffer(N)}_. The ArrayBuffer is an image and can be 'large', several MBytes. When I export this table, a json is created, no problem. But if I try to import the json, an exception occurs:  **_Error: Max buffer length exceeded: textNode_**

I get the error with only one entry in my table, the export file is about 3MB. I guess it is not directly a Dexie issue but I hope I am not the first one to face this kind of problem 

Thanks !!

**Comment by tomgallagher** [comment]
Yes, I'm getting this as well with a data-uri image string. I'm thinking about filtering the image out when exporting but I'd prefer not to.

**Comment by tomgallagher** [comment]
This is an upstream problem with Clarinet

https://github.com/dscape/clarinet/issues/38

**Comment by dfahlander** [comment]
I've left a comment inte clarinet issue and did a fast-fix in my clarinet fork to allow 10 MB instead of 64 kb. If not enough then could increase even more. I think we will not come away from having to store each JSON string into memory. At a certain point of time, the row has to be put somewhere so the least piece of data we can store is a single row and if that row must have some kind of limit.

**Comment by codeapologist** [comment]
Hi, my project has run into the buffer limit even with the increase to 10MB.  A significant portion of our app operates offline, and we store a considerable amount of data in Dexie (export file of 150MB+).  I have pulled down the project and tweaked the buffer size to perform some analysis with our datasets. We think a reasonable max buffer length for our use case is 30MB.  Is it possible to increase this to 30MB?  Thanks in advance!
